### ISSUE TITLE
Certain learning rate decay step

### ISSUE THREAD

**Issue by chasewinds** [brucechou1983/CheXNet-Keras#34]
The implementation of learning rate decay every time validation loss stop decrease is vary elegant in the code, but unfortunately I use native tensorflow to implemente CheXNet instead of Keras, which means I must  write more code for that function.
Now I just want to recurrent the result of CheXNet as a baseline so I would be appreciate if anyone can provide a gourp of hyperparameters include learning rate decay step and correspond learning rate which lead to a well trained model.
My code can receive decay step and learning rate in the following format so I'm sure I can train my model preparly if good learning rate is provide:
epochs_lr = [[5, 0.001], [20, 0.0001], [10, 0.00001], [5, 0.000001]]
